spec/system/bitklavier_single_item_metadata_spec.rb

```diff
@@ -31,7 +31,7 @@
   it "has expected metadata" do
     visit '/catalog/doi-10-34770-r75s-9j74'
     expect(page).to have_content "bitKlavier Grand Sample Library—Binaural Mic Image"
-    authors = "<span>Trueman, Daniel; Wang, Matthew; Villalta, Andrés; Chou, Katie; Ayres, Christien</span>"
+    authors = "<span>Trueman, Daniel</span>"
```

Comment:
The new fixture has only one author (the rest are now contributors and that's tested on the indexing spect)